### PR TITLE
Add rule for SLES-15-010356

### PR DESCRIPTION
**Pull request by vtrubovics**
**Description:**

-  Add rule to 'Verify that system-wide directories Have Root Group Ownership'

**Rationale:**

System-wide directories should be group-owned by root.

- Add oval check for directories containing system library files having correct ownership
- Add ansible remediation based on combination of find and file modules in order to avoid lint warnings for file permissions when using find shell command and have no previous knowledge on the directories permission modes
- Add couple of tests to verify pass and fail behaviour
- Add rule for SLES-15-010356 to stig profile

### CODE REVIEW

**Comment by ggbecker** [comment]
@openscap-ci ok to test

File: linux_os/guide/system/permissions/files/dir_syslibraries_root_owned_group/rule.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    and "/usr/lib64" are group-owned by root.
+
+description: |-
+    System-wide library directories are:
+    <pre>/lib
+    /lib64
+    /usr/lib
+    /usr/lib64
+    </pre>
+    All system-wide directories should be protected from unautorised
```
Comment (ggbecker) [inline linux_os/guide/system/permissions/files/dir_syslibraries_root_owned_group/rule.yml:16]:
```suggestion
    All system-wide directories should be protected from unauthorised 
```

Comment (vtrubovics) [reply linux_os/guide/system/permissions/files/dir_syslibraries_root_owned_group/rule.yml]: Fixed.

**Comment by ggbecker** [comment]
@openscap-ci add to whitelist

**Pull request closed without merging**